x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,124 @@
+import { useKibana } from '../../../../../common/lib/kibana';
+
+const CenteredEuiFlexItem = styled(EuiFlexItem)`
+  align-items: center;
+`;
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  const { docLinks } = useKibana().services;
+  const { size } = useEuiTheme().euiTheme;
+
+  const title = (
+    <EuiText>
+      <h3>
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.title"
+          defaultMessage="We'll save your integration with our recommended defaults."
+        />
+      </h3>
+    </EuiText>
+  );
+
+  const logoSize = `calc(2 * ${size.xxxxl})`;
+  const securityLogo = (
+    <EuiIcon
+      type="logoSecurity"
+      css={css`
+        width: ${logoSize};
+        height: ${logoSize};
+      `}
+    />
+  );
+
+  const features = (
+    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiIcon type="check" />
+      </EuiFlexItem>
+
+      <EuiFlexItem>
+        <EuiText size="m">
+          <p>
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.feature"
+              defaultMessage="Windows, Mac, and Linux event collection"
+            />
+          </p>
+        </EuiText>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  const details = (
+    <EuiText size="m" color="subdued">
+      <p>
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.details"
+          defaultMessage="You can change this later by editing the Endpoint Security integration agent policy.
+      Read more about Endpoint security configuration in our {docsPage}."
+          values={{
+            docsPage: (
+              <EuiLink
+                href={docLinks.links.securitySolution.configureEndpointIntegrationPolicy}
+                target="_blank"
+                external
+              >
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.docsPage"
+                  defaultMessage="documentation"
+                />
+              </EuiLink>
+            ),
+          }}
+        />
+      </p>
+    </EuiText>
+  );
+
+  const marginSize = `calc(${size.m} + ${size.l})`;
+
+  return (
+    <EuiPanel hasShadow={false} paddingSize="l">
+      <EuiSpacer size="xl" />
+
+      {title}
+
+      <EuiFlexGroup
+        css={css`
+          margin-bottom: ${marginSize};
+          margin-top: ${marginSize};
+        `}
+      >
+        <CenteredEuiFlexItem grow={false}>{securityLogo}</CenteredEuiFlexItem>
```

Comment:
I don't think you need this additional center alignment. You can just use the `EuiFlexItem`.

Reply:
For the given design, you are right! The center alignment helps with the mobile view.